smarts/core/chassis.py

```diff
@@ -229,35 +229,50 @@ def __init__(
         assert isinstance(pose, Pose)
         self._log = logging.getLogger(self.__class__.__name__)
 
+        if vehicle_filepath is None:
+            with pkg_resources.path(models, "vehicle.urdf") as path:
+                vehicle_filepath = str(path.absolute())
+
+        if controller_parameters_filepath is None:
+            with pkg_resources.path(
+                models, "controller_parameters.yaml"
+            ) as controller_path:
+                controller_filepath = str(controller_path.absolute())
+            with open(controller_filepath, "r") as controller_file:
+                controller_parameters_filepath = yaml.safe_load(controller_file)["car"]
```

Comment:
We resolve these paths when we build the AckermanChassis in `vehicle.py`, I think we can remove these.